pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -686,8 +718,17 @@ func (h *Helper) updateConfig(configDir string, opt *StartOptions) error {
 	if err != nil {
 		return err
 	}
-	nodeCfg.DNSBindAddress = ""
-	nodeCfg.DNSIP = ""
+	version, err := h.serverVersion()
+	if err != nil {
+		return err
+	}
+	if useBindAddress(version) {
+		nodeCfg.DNSBindAddress = "127.0.0.1:53"
+		nodeCfg.DNSIP = "172.30.0.1"
```

Comment:
and I don't have to check the version, since the previous versions just ignore the DNSIP